src/utils/IdealGraphVisualizer/Data/src/main/java/com/sun/hotspot/igv/data/serialization/Parser.java

```diff
@@ -145,12 +146,17 @@ protected Group start() throws SAXException {
 
             final Folder parent = getParentObject();
             if (groupCallback == null || parent instanceof Group) {
-                SwingUtilities.invokeLater(new Runnable(){
-                    @Override
-                    public void run() {
-                        parent.addElement(group);
-                    }
-                });
+                Runnable addToParent = new Runnable(){
+                        @Override
+                        public void run() {
+                            parent.addElement(group);
+                        }
+                    };
```

Comment:
You could directly use `Runnable addToParent = () -> parent.addElement(group);` and also for the other ones below.

Reply:
Good suggestion, thanks, done!